pkgs/build-support/setup-hooks/wrap-gapps-hook/default.nix

```
@@ -27,16 +28,22 @@ makeSetupHook {
     # TODO: remove this, packages should depend on GTK explicitly.
     gtk3
 
-    # librsvg provides a module for gdk-pixbuf to allow rendering
-    # SVG icons. Most icon themes are SVG-based and so are some
-    # graphics in GTK (e.g. cross for closing window in window title bar)
-    # so it is pretty much required for applications using GTK.
     librsvg
   ] ++ [
 
     # We use the wrapProgram function.
     makeWrapper
   ];
+
+  # depsTargetTargetPropagated will be buildInputs
```

Was this meant to be something like

```suggestion
  # depsTargetTargetPropagated will essentially be buildInputs when wrapGAppsHook is placed into nativeBuildInputs
```